devel/jreleaser/Portfile

```diff
@@ -1,15 +1,15 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# Generated with JReleaser 1.4.0-SNAPSHOT at 2022-11-05T10:10:05.154313676Z
+# Generated with JReleaser 1.4.0-SNAPSHOT at 2022-12-29T16:04:33.910943262Z
 
 PortSystem       1.0
 PortGroup        github 1.0
 PortGroup        java 1.0
 
-github.setup     jreleaser jreleaser 1.3.1 v
+github.setup     jreleaser jreleaser 1.4.0 v
 revision         0
 
 categories       devel java
-license          Apache-2
+license          Apache-2.0
 maintainers      @aalmiray
```

Comment:
Is there a particular reason why this port isn't also `openmaintainer`? Just curious.

Reply:
This port is automatically generated by JReleaser itself based on a template file (see https://github.com/jreleaser/jreleaser/blob/main/src/jreleaser/distributions/jreleaser/macports/Portfile.tpl). Any changes required by the port should be made to that template file as well. This port is updated on every release, roughly every 2 months.

Reply:
Ah OK, gotcha.